roles/zuul/files/jobs/hoist.yaml

```diff
@@ -10,7 +10,7 @@
           #!/bin/bash -ex
           source /opt/venvs/ansible/bin/activate
           ansible-playbook -i inventory/nodepool.py install-ci.yml --skip-tags monitoring -e @secrets.yml.example
-          ansible-playbook -i inventory/nodepool.py tests/validate-ci.yml
+          ansible-playbook -i inventory/nodepool.py tests/files/validate-ci.yml
```

Comment:
So this entire file can be dropped. We run all this stuff in repo now, in hoist.git/tests/